src/commands/source/download.rs

```diff
@@ -305,13 +305,12 @@ pub async fn download(
 
     if r.is_err() {
         progressbar.lock().await.error().await;
+        return r;
     } else {
         progressbar.lock().await.success().await;
     }
```

Comment:
Oops, I could've dropped that else once I added the return but, while it's unnecessary, I kinda like it visually...